docs/tutorial/tutorial-1.rst

```diff
@@ -19,8 +19,10 @@ that by writing in your current terminal::
 
     VOC_DIR=/path/to/voc
 
-In case you don't really want to do that for some reason, you'll need to
-remember to replace ``$VOC_DIR`` by the full VOC directory any time it
+Please remember to replace ``/path/to/voc`` with the actual path to your VOC 
+repository (e.g. /home/elias/src/voc-dev/voc). In case you don't really want 
```

Comment:
Two small fixes here - firstly, replace `elias` with something generic like `myusername`; secondly, since this is a literal, it should be wrapped with backticks.

Reply:
thanks for the feedback, much appreciated!